src/apps/vpn/platforms/ios/iosconstants.swift

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+struct Constants {
+  static let appGroupIdentifier = "group.org.mozilla.ios.Guardian"
+
+  struct UserDefaultKeys {
+    static let telemetryEnabled = "TELEMETRY_ENABLED"
+    static let appChannel = "APP_CHANNEL"
```

Comment:
"APP_CHANNEL"? Should this maybe be release?

Reply:
This is the key, not the value.